src/display.py

```diff
@@ -62,6 +65,21 @@ def update(self, event):
             if   event.key == pygame.K_BACKSPACE: self.text = self.text[:-1]
             elif event.unicode.isdigit()        : self.text += event.unicode
         
+class DisplayBox(InputBox):
```

Comment:
I'm not sure sure `DisplayBox` is a good name, all boxes display something.
Looking at the other names `TextBox`, `SlideBox`, `DropdownBox`, ...
Maybe `TimeBox` or something like that would be better.